### ISSUE TITLE
Is it ok to use google API

### ISSUE THREAD

**Issue by Tonku** [lennylxx/google-input-tools-macos#6]
Thanks for the library...
I am just wondering, can I use this API for a commercial application?

**Comment by lennylxx** [comment]
I don't think so, maybe you need to have a special agreement with Google to use it for commercial.
